circuit-codegen-annotations/build.gradle.kts

```diff
@@ -86,4 +86,11 @@ kotlin {
   }
 }
 
-android { namespace = "com.slack.circuit.codegen.annotations" }
+android {
+  namespace = "com.slack.circuit.codegen.annotations"
+  defaultConfig {
+    consumerProguardFiles(
+      "src/commonJvmMain/resources/META-INF/proguard/circuit-codegen-annotations.pro"
+    )
+  }
```

Comment:
This is not necessary, the jar is enough for both

Reply:
Without this `consumerProguardFiles(...)` I'm still getting the `ERROR: R8: Missing class`. 

The dependency we use in our android project is `"com.slack.circuit:circuit-codegen-annotations"` so I believe it fallback on the `-android` architecture (aar), not `-jvm` (jar). I can see the `proguard.txt` files in the jar but not in the aar (if I use `"com.slack.circuit:circuit-codegen-annotations-jvm"` the build passes).

Or Am I missing something?

Reply:
iiintersting, didn't know that they didn't support it from aar artifacts. Good to know 👍